### DISCUSSION TITLE
Running Interactsh ephemerally / on demand

### DISCUSSION THREAD

**Discussion by Bwvolleyball** [projectdiscovery/interactsh#314] [Ideas]
### Please describe your feature request:

Add the ability / option to persist runtime state (certificates, DNS names, static IP addresses, hits) externally

### Describe the use case of this feature:

In a situation where a user wants to run interactsh as an on demand or scale-to-0 application, having all runtime state persisted externally (to an external cache such, or to an external file system for certificates), would allow for running interactsh on Kubernetes / Knative and allow for quick and consistent reboots of interactsh, in the event the application is scaled down between runs, or a pod died and was rescheduled.

**Comment by MetzinAround** [comment]
Closing 1.5 year old issue. Feel free to reopen if needed.
